staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2640,17 +2640,28 @@ func (i *IngressDescriber) describeIngressV1(ing *networkingv1.Ingress, events *
 		if def == nil {
 			// Ingresses that don't specify a default backend inherit the
 			// default backend in the kube-system namespace.
-			def = &networkingv1.IngressBackend{
+			defaultBackend := &networkingv1.IngressBackend{
 				Service: &networkingv1.IngressServiceBackend{
 					Name: "default-http-backend",
 					Port: networkingv1.ServiceBackendPort{
 						Number: 80,
 					},
 				},
 			}
-			ns = metav1.NamespaceSystem
+			_, err := i.client.CoreV1().Endpoints(metav1.NamespaceSystem).Get(context.TODO(), defaultBackend.Service.Name, metav1.GetOptions{})
```

Comment:
That's not what I requested, in https://github.com/kubernetes/kubernetes/issues/107720#issuecomment-1058224411 I've meant to only print `<default>` when we there is no default backend specified without checking for anything. As mentioned in https://github.com/kubernetes/kubectl/issues/1172#issuecomment-1026135953 it's a convention name, so I'd prefer to only print `<default>` in those cases.

Reply:
Sorry about that. Fixed.